### PR TITLE
Add subsubsection to explain how to change the path when the position…

### PR DESCRIPTION
**Pull request by trojanobelix**
… of the submodule has changed. See: Moving or changing the path of submodules #2222

### CODE REVIEW

**Comment by RussKie** [comment]
@gerhardol may I get you to review please?

File: source/submodules.rst [hunk 1]
```diff
@@ -30,6 +30,11 @@ the list on the left.
 |Remove        | Remove the submodule from the repository                                                                        |
 +--------------+-----------------------------------------------------------------------------------------------------------------+
 
+Change submodule path
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+Delete the existing submodule and add it again in the new location after changing the submodule location.
+
+
```
Comment (gerhardol) [inline source/submodules.rst:37]: Suggest not adding a header (the ^^^^^ length should match the text exactly if kept) and to reword a little.
```suggestion
To change a submodule path, delete the existing submodule, move the filesystem directory and add it again in the new location.

```

**Review by gerhardol — COMMENTED** [review]
+1

**Comment by gerhardol** [comment]
@trojanobelix Can we apply the changes and squash merge this PR?

**Comment by trojanobelix** [comment]
Yes, of cause. Thanks.

**Comment by gerhardol** [comment]
Changing in #139

**Pull request closed without merging**